internal/binutils/binutils.go

```diff
@@ -159,10 +183,19 @@ func findExe(cmd string, paths []string) (string, bool) {
 // of a binary.
 func (bu *Binutils) Disasm(file string, start, end uint64, intelSyntax bool) ([]plugin.Inst, error) {
 	b := bu.get()
+	if !b.objdumpFound {
+		return nil, fmt.Errorf("couldn't find objdump")
+	}
 	args := []string{"-d", "-C", "--no-show-raw-insn", "-l",
 		fmt.Sprintf("--start-address=%#x", start),
 		fmt.Sprintf("--stop-address=%#x", end)}
 
+	if runtime.GOOS == "darwin" {
+		args = []string{"-disassemble-all", "-no-show-raw-insn",
```

Comment:
Avoid re-assignment, this makes code more difficult to understand. Initialize once:
```
var foo ...
if (...) {
  foo = ...
} else {
  foo = ...
}
```

Reply:
Two-dash `--no-show-raw-insn` should work as well, for all llvm-objdump versions I know. Is it not the case for macOS objdump?